website/docs/cloud-docs/api-docs/applies.mdx

```diff
@@ -174,3 +152,50 @@ _Using Terraform Cloud Agents_
   }
 }
 ```
+
+## Recover a failed state upload
+
+`GET /applies/:id/errored-state`
```

Comment:
I don't think the nesting is necessary. In general the API, being a JSON:API flavored structure, favors less nesting on the API endpoints, opting for wholly separate top-level objects instead. These errored states are a type of exception because they aren't represented like a true StateVersion is - the errored state is never parsed and doesn't require a dedicated record in the database to represent it. It's just an object in our backend storage that we're providing the user access to.